datacapture/src/main/java/com/google/android/fhir/datacapture/views/QuestionnaireItemAttachmentViewHolderFactory.kt

```diff
@@ -124,8 +124,23 @@ internal object QuestionnaireItemAttachmentViewHolderFactory :
         deleteButton.isEnabled = !isReadOnly
       }
 
-      private fun displayInitialPreview() {
-        questionnaireItemViewItem.answers.firstOrNull()?.valueAttachment?.let { attachment ->
+      private fun displayInitialPreviewOrClear() {
```

Comment:
this sounds better:

```suggestion
      private fun displayOrClearInitialPreview() {
```

or actually the original function name seemed fine to me.

up to you

Reply:
i actually like this one

Reply:
the new name i mean